oag/ja-jp/content/topics/releasenotes/oag/oag-version-history.htm

```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```

Comment:
Drat, this is the older version, from a previous template. Will need to update this here and on the LL branch.

Reply:
If PR does not include additional commits, It's safer to update LL branch and re-run the receiving script